runelite-client/src/main/java/net/runelite/client/plugins/chatcommands/ChatCommandsPlugin.java

```diff
@@ -629,6 +645,11 @@ private void playerSkillLookup(ChatMessage chatMessage, String message)
 		{
 			return;
 		}
+		
+		if (message.length() <= LEVEL_COMMAND_STRING.length())
```

Comment:
Even though I think this works, this should be moved into the case for message starting with `LEVEL_COMMAND_STRING`

Reply:
Oh because it is also called when you do !total. I see, missed that initially. Thanks.